### PR TITLE
Improved "Latest Recommendation" link in the document header

### PR DESCRIPTION
**Pull request by himorin**
***
<a href="https://pr-preview.s3.amazonaws.com/himorin/imsc/pull/538.html" title="Last updated on Apr 7, 2020, 4:32 PM UTC (1e36b32)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/imsc/538/6a0bdfe...himorin:1e36b32.html" title="Last updated on Apr 7, 2020, 4:32 PM UTC (1e36b32)">Diff</a>

### CODE REVIEW

File: imsc1/spec/ttml-ww-profiles.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <head>
   <meta charset='utf-8'>
 
-  <title>TTML Profiles for Internet Media Subtitles and Captions 1.1</title>
+  <title>TTML Profiles for Internet Media Subtitles and Captions 1.1 (Link corrected 9 April 2020)</title>
```
Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html:7]: Do we need to add this change @plehegar ?

File: imsc1/spec/ttml-ww-profiles.html [hunk 2]
```diff
@@ -26,7 +26,7 @@
                 ,   otherLinks: [{
                           key: 'Latest IMSC1 recommendation',
```
Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html:27]:
```suggestion
                          key: 'Latest IMSC recommendation',
```

File: imsc1/spec/ttml-ww-profiles.html (hunk 1 above)
Comment (palemieux) [reply imsc1/spec/ttml-ww-profiles.html:7]: Let's not change the title, and instead insert a NOTE, as was done in the past: https://www.w3.org/TR/2016/REC-ttml-imsc1-20160421/

Comment (plehegar) [reply imsc1/spec/ttml-ww-profiles.html:7]: +1 for not changing the title

File: imsc1/spec/ttml-ww-profiles.html (hunk 2 above)
Comment (plehegar) [reply imsc1/spec/ttml-ww-profiles.html:27]: SGTM

File: imsc1/spec/ttml-ww-profiles.html (hunk 1 above)
Comment (plehegar) [reply imsc1/spec/ttml-ww-profiles.html:7]: Use instead soimething like:
[[
W3C Recommendation 21 April 2016
(Link corrected 9 April 2020)
]]

and additional note can go into the SOTD.

Comment (nigelmegitt) [reply imsc1/spec/ttml-ww-profiles.html]: I would prefer a note to this change.

File: imsc1/spec/ttml-ww-profiles.html [hunk 3]
```diff
@@ -208,6 +208,9 @@
   </section>
 
   <section id='sotd'>
+    <p class="note">
+        Status Update (April 2020): The link to the latest IMSC1 Recommendation in the header was fixed in-place on April 9 2020: they were pointing to TTML IMSC 1.0.1, and were fixed to link to the latest Recommendation of TTML IMSC.
```
Comment (palemieux) [inline imsc1/spec/ttml-ww-profiles.html:212]: `Status Update (9 April 2020): In the document header, the "Latest IMSC1 Recommendation" link (pointing to the specific "TTML Profiles for Internet Media Subtitles and Captions 1.0.1" Recommendation) was replaced by a "Latest IMSC Recommendation" link pointing to the most recently published Recommendation in the "TTML Profiles for Internet Media Subtitles and Captions" family of Recommendations.`

Comment (himorin) [reply imsc1/spec/ttml-ww-profiles.html]: thank you for suggestion. text updated.

**Comment by himorin** [comment]
take down this PR, no way to use respec for in-place modification.

**Pull request closed without merging**